app/uk/gov/hmrc/cgtpropertydisposalsfrontend/services/returns/ReturnsService.scala

```diff
@@ -173,6 +181,94 @@ class ReturnsServiceImpl @Inject() (
                  else EitherT.rightT[Future, Error](())
     } yield unsentDraftReturnsWithTaxYearExchangedAndSAStatus
 
+  def updateCorrectTaxYearToSentReturns(
```

Comment:
Shall this be names as "updateCorrectTaxYearToDraftReturns"?

Reply:
They are sent returns only as we are referring `ReturnSummary`. We dont need to update "taxYear" field to draft returns.